plugin/vscode/performance-writer.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2020  Matthew Glazar
+// See end of file for extended copyright information.
+
+const fs = require('fs');
+const path = require("path");
+const process = require("process");
+
+class PerformanceWriter {
+    
+    constructor (extensionContext) {
+        this._dataAttributes = {
+            columns:'timestamp,file-ref,file-name-or-duration,event-id'
+        };
+        this._tmpLogLines = []
+        this._fileName = null;
+        this._fileIndex = null;
+        this._extensionContext = extensionContext;
+        fs.mkdirSync(this._extensionContext.logUri.fsPath)
```

Comment:
Performance: I'd rather this be asynchronous. Perhaps do this in `writeToFile`? (You'd need to lazily execute `fs.createWriteStream` in `writeToFile`.)

Reply:
[Not addressed in latest commit]
Let's discuss this more.